hooks/open-telemetry/pkg/traces.go

```diff
@@ -81,4 +83,11 @@ func WithErrorStatusEnabled() Options {
 	}
 }
 
-var _ openfeature.Hook = &traceHook{}
+// WithAttributeSetterForTraces allows to set a extractionCallback which accept openfeature.FlagMetadata and returns
+// []attribute.KeyValue derived from that metadata.
+// If present, returned attributes will be added to successful evaluation traces
+func WithAttributeSetterForTraces(callback *func(openfeature.FlagMetadata) []attribute.KeyValue) Options {
```

Comment:
see https://github.com/open-feature/go-sdk-contrib/pull/311/files#r1291638254

Reply:
Renamed to `WithTracesAttributeSetter`